packages/extension/src/shared/call/changeMultisigSignersCall.ts

```diff
@@ -17,3 +17,21 @@ export const isAddMultisigSignersCall = (
   }
   return false
 }
+
+export interface RemoveMultisigSignersCall extends Call {
+  entrypoint: "removeSigners"
+}
+
+export const isRemoveMultisigSignersCall = (
+  call: Call,
+): call is AddMultisigSignersCall => {
+  try {
+    if (call.contractAddress && call.entrypoint === "removeSigners") {
+      validateAndParseAddress(call.contractAddress)
```

Comment:
```suggestion
      validateAndParseAddress(call.contractAddress)
```
This seems better https://github.com/argentlabs/argent-x/blob/bb5d62371cedfa90f6876f207985b74a1d9515c2/packages/extension/src/ui/services/addresses.ts#L76

Reply:
I saw this in the 2fa calls files, I'm indifferent about either but I think we should only have one way of checking this